Sources/NSAttributedStringExtensions.swift

```diff
@@ -73,5 +73,12 @@ public func += (left: inout NSAttributedString, right: NSAttributedString) {
     ns.append(right)
     left = ns
 }
+    
+/// EZSE: Sum of one NSAttributedString with another NSAttributedString
+public func + (left: NSAttributedString, right: NSAttributedString) -> NSAttributedString {
```

Comment:
Yes. It [does currently support **+=**](https://github.com/goktugyil/EZSwiftExtensions/blob/adding_attributed_string_extension/Sources/NSAttributedStringExtensions.swift#L71)